api/s2n.h

```diff
@@ -2879,6 +2915,18 @@ S2N_API extern int s2n_connection_get_actual_protocol_version(struct s2n_connect
  */
 S2N_API extern int s2n_connection_get_client_hello_version(struct s2n_connection *conn);
 
+/**
+ * Access the Client Hello protocol version written in the Client Hello
+ * record header.
```

Comment:
This isn't really right-- it's not the "Client Hello protocol version", it's the record protocol version used for the record containing the ClientHello.
```suggestion
 * Access the protocol version found in the ClientHello record header
```